src/models/Expense.model.js

```diff
@@ -1,9 +1,43 @@
 'use strict';
 
 const { sequelize } = require('../db.js');
+const { DataTypes } = require('sequelize');
 
 const Expense = sequelize.define(
-  // your code goes here
+  'Expense',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+    },
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    spentAt: {
+      type: DataTypes.TEXT,
+      allowNull: false,
+    },
+    title: {
+      type: DataTypes.TEXT,
+      allowNull: false,
+    },
+    amount: {
+      type: DataTypes.INTEGER,
```

Comment:
The 'amount' field is using DataTypes.INTEGER. Consider using DataTypes.DECIMAL or DataTypes.FLOAT if you need to handle fractional amounts or larger numbers.